sdk/canton/component.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: digitalasset.com/v1
```

Comment:
@samuel-williams-da can't you just pull down the already published canton artifact rather than building a separate one here?